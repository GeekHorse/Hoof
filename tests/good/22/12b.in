new right foo oof goo done
new down bar rab done
new down baz zab done
most up
dig foo oof cancel
value
save
