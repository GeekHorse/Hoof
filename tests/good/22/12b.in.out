new right foo oof goo done
new down bar rab done
new down baz zab done
